ros_ws/src/simulation/simulation_tools/src/simulation_tools/lap_timer.py

```diff
@@ -61,6 +64,12 @@ def complete_lap(self):
             rospy.loginfo("Lap " + str(len(self.history)) + " (" + self.name + "): " +  # nopep8
                           format_duration(duration) + ", average: " + format_duration(average))  # nopep8
         self.start = time
+        self.msg_seq += 1
+        pub_msg = DurationStamped()
```

Comment:
Consider renaming this to `message` or `duration_message`.

Reply:
Done.